smurfs/src/components/Smurfs.js

```diff
@@ -0,0 +1,36 @@
+import React, {useState, useEffect} from 'react';
+import { connect } from "react-redux";
+import  * as actionCreators  from "../state/actionCreators";
+
+export function Smurf({ getSmurfs, smurfs }) {
+    useEffect(() => {
+      getSmurfs();
+    }, []);
```

Comment:
```suggestion
    }, []);
      if (!smurfs) return null
```

Reply:
This will resolve the issue that crashes the application when you add new smurfs